hedy.py

```diff
@@ -1367,7 +1367,8 @@ def merge_grammars(grammar_text_1, grammar_text_2):
 def create_grammar(level):
     # start with creating the grammar for level 1
     result = get_full_grammar_for_level(1)
-
+    keys = get_keywords_for_language("en")
+    result = merge_grammars(result, keys)
     # then keep merging new grammars in
     for i in range(2, level+1):
         grammar_text_i = get_additional_rules_for_level(i)
```

Comment:
The grammar saving below will need to differentiate between keyword languages I think.

Reply:
Also I think the grammar cache needs to consider keyword language.

Reply:
Yes @bjorn3 is right, the cache now indexes with level number only, that will need to be updated to level + lang (for example: "en-1" and "nl-7")

Reply:
Thanks for your feedback! I have now made a commit ('Resolving comments on pull request') which hopefully resolves these issues. If not, please let me know.